### PR TITLE
add adelais en ciel

### PR DESCRIPTION
**Pull request by mechlovin**
## Description

Add Adelais en ciel - compatible with TGR Alice and all clone variants
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mechlovin/adelais_en_ciel/rules.mk [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+# If you don't know the bootloader type, then you can specify the
+# Boot Section Size in *bytes* by uncommenting out the OPT_DEFS line
+# Otherwise, delete this section
+#   Teensy halfKay      512
+#   Teensy++ halfKay    1024
+#   Atmel DFU loader    4096
+#   LUFA bootloader     4096
+#   USBaspLoader        2048
+# OPT_DEFS += -DBOOTLOADER_SIZE=4096
```
Comment (fauxpark) [inline keyboards/mechlovin/adelais_en_ciel/rules.mk:13]:
```suggestion
```

Comment (mechlovin) [reply keyboards/mechlovin/adelais_en_ciel/rules.mk]: tks u

File: keyboards/mechlovin/adelais_en_ciel/adelais_en_ciel.c [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "adelais_en_ciel.h"
+led_config_t g_led_config = { {
```
Comment (drashna) [inline keyboards/mechlovin/adelais_en_ciel/adelais_en_ciel.c:18]:
```suggestion

#ifdef RGB_MATRIX_ENABLE
led_config_t g_led_config = { {
```

File: keyboards/mechlovin/adelais_en_ciel/adelais_en_ciel.c [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+}, {
+  1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,            1,           1,           1,
+  1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,            1,           1,           1,  
+  1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,            1,           1,           1,  
+  1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,            1,           1,           1,  
+  1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1,          1, 
+} };
```
Comment (drashna) [inline keyboards/mechlovin/adelais_en_ciel/adelais_en_ciel.c:39]:
```suggestion
} };
#endif
```

File: keyboards/mechlovin/adelais_en_ciel/adelais_en_ciel.c (hunk 2 above)
Comment (mechlovin) [reply keyboards/mechlovin/adelais_en_ciel/adelais_en_ciel.c:18]: this pcb is rgb pcb, so i think we don't need need to define RGB_MATRIX_ENABLE

Comment (drashna) [reply keyboards/mechlovin/adelais_en_ciel/adelais_en_ciel.c:18]: It is, but somebody may want to ... *shudders* disable the RGB matrix on the board. 

And if they do, the compile will fail if this isn't behind a preprocessor ifdef.

**Comment by mechlovin** [comment]
Thank you! Have to close this PR.

**Pull request closed without merging**